File: algorithms/bubble_sort.py

def sort(list):
    tmp = 0
    for i in range(len(list)):
        for j in range(len(list) - 1, i, -1):
            if list[j-1] > list[j]:
                tmp = list[j]
                list[j] = list[j-1]
                list[j-1] = tmp

def main():
    list = [ 4, 3, 1, 5, 2, 6 ]
    sort(list)
    print(list)

if __name__ == "__main__":
    main()

File: algorithms/bubble_sort.js

function sort(list) {
    for (let i = 0; i < list.length; i++) {
        for (let j = list.length - 1; j > i ; j--) {
            if (list[j-1] > list[j]) { 
                var tmp;
                tmp = list[j-1];
                list[j-1] = list[j];
                list[j] = tmp;
            }  
        }
    }    
}

(() => {
    const list = [3, 2, 4, 1, 6, 5];
    sort(list);
    console.log(list); 
})();